_episodes/32-software-architecture-design.md

```diff
@@ -130,19 +130,20 @@ either as a whole or in terms of its individual functions.
 Now that we know what goals we should aspire to, let us take a critical look at the code in our 
 software project and try to identify ways in which it can be improved. 
 
-Our software project contains a branch `full-data-analysis` with code for a new feature of our 
-inflammation analysis software. Recall that you can see all your branches as follows:
+Our software project contains a pre-existing branch `full-data-analysis` which contains code for a new feature of our 
+inflammation analysis software, which we'll consider as a contribution by another developer.
+Recall that you can see all your branches as follows:
 
 ~~~
 $ git branch --all
 ~~~
 {: .language-bash}
 
-Let's checkout a new local branch from the `full-data-analysis` branch, making sure we
-have saved and committed all current changes before doing so.
+Once you have saved and committed any current changes,
+checkout this `full-data-analysis` branch:
 
 ~~~
-git checkout -b full-data-analysis
+git checkout full-data-analysis
```

Comment:
Think we switched to using the 'git switch' command instead of 'git checkout' - Thomas was probably not aware.

Reply:
Yep, change to switch1

Reply:
Yep change to switch!